### ISSUE TITLE
Lohit Gujarati font's English Character & Number should be changed

### ISSUE THREAD

**Issue by nileshbandhiya** [pravins/lohit#69]
Lohit Gujarati font working good.
I am facing the problem when typing non-gujarati character, it seen unreadable.

Specially, when we are using a numbers while typing Gujarati in Lohit Gujarati font, it can not render properly.

INR symbol should be update as thinner than it appear currently

Kindly check below:
![english_number_and_rupee_sign](https://cloud.githubusercontent.com/assets/5889363/7132653/cd3b1d9a-e2a8-11e4-83c2-3c18181538f1.PNG)

**Comment by nileshbandhiya** [comment]
Although, if we select any other font for Latin/English script, there is no issue.
But whole document is selected with Lohit Gujarati, then it happen.
![custom_font](https://cloud.githubusercontent.com/assets/5889363/7132814/6858a698-e2aa-11e4-9b11-6aab331cad6d.PNG)

**Comment by pravins** [comment]
In Lohit Gujarati we dont have English characters presently. Presently whenever one type English characters renderer mostly takes shapes from default font of the system. 
We recently added in Devanagari http://pravin-s.blogspot.in/2014/11/next-improved-release-of-lohit.html  
If someone help, would like to do same for Gujarati as well.

**Comment by nileshbandhiya** [comment]
Yes, We can add Latin characters into Lohit Gujarati for proper rendering.
But, is it official font from Lohit - "Lohit Latin"? Becuase, it is not listed at https://fedorahosted.org/lohit/ and it may developed/maintend by user "etunni" at GitHub.

And shape for ₹ (INR) should be update.
Thanks Pravinbhai... Regards,
Nilesh Bandhiya
 


     On Tuesday, 14 April 2015 3:18 PM, Pravin Satpute <notifications@github.com> wrote:
   

 In Lohit Gujarati we dont have English characters presently. Presently whenever one type English characters renderer mostly takes shapes from default font of the system. 
We recently added in Devanagari http://pravin-s.blogspot.in/2014/11/next-improved-release-of-lohit.html

If someone help, would like to do same for Gujarati as well. —

**Comment by davelab6** [comment]
@pravins when will you integrate https://github.com/etunni/lohit-latin ? :)

**Comment by pravins** [comment]
As pointed earlier, we might need minor tweak (scaling etc) while integrating in Lohit Gujarati. Will be nice if you can help :)

**Comment by nileshbandhiya** [comment]
Yes. I am ready to help. What should I do?
